scripts/west_commands/runners/nrf_common.py

```diff
@@ -92,11 +86,6 @@ def do_add_parser(cls, parser):
                             help='''erase all user available non-volatile
                             memory and disable read back protection before
                             flashing (erases flash for both cores on nRF53)''')
-        parser.add_argument('--erase-all-uicrs', required=False,
```

Comment:
What would be the procedure to erase unused core?
I.e.. There was fw written at APP core (empty RAD), then I want to run application at Radio core.
How can I prevent execution at APP core?
Previously this was done by this arg = --erase-all-uicrs